.github/workflows/ci-github-actions-self-hosted.yaml

```diff
@@ -293,3 +293,100 @@ jobs:
           state: ${{job.status}}
           sha: ${{fromJson(steps.request.outputs.data).head.sha}}
           target_url: https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
+
+  nitrogen2-amdclang:
+    if: |
+      github.repository_owner == 'QMCPACK' &&
+      github.event.issue.pull_request &&
+      ( startsWith(github.event.comment.body, 'Test this please') || 
+        startsWith(github.event.comment.body, 'Start testing in-house') )
+
+    runs-on: [self-hosted, Linux, X64, ornl-nitrogen2]
+
+    env:
+      GH_JOBNAME: ${{matrix.jobname}}
+      GH_OS: Linux
+    strategy:
+      fail-fast: false
+      matrix:
+        jobname:
+          [
+            MI210-AMDClang-NoMPI-Offload-Real-Mixed,
+            MI210-AMDClang-NoMPI-Offload-Real,
+            MI210-AMDClang-NoMPI-Offload-Real-Mixed,
+            MI210-AMDClang-NoMPI-Offload-Real,
```

Comment:
Real was repeated. The last two should be complex. I will add a fix via https://github.com/QMCPACK/qmcpack/pull/5681

Reply:
Done.